packages/node_modules/samples/browser-auth-implicit/test/wdio/spec/implicit-grant-authentication.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable no-undef */
+
+import testUsers from '@ciscospark/test-helper-test-users';
+import {assert} from 'chai';
+
+describe(`samples/browser-auth-implicit`, () => {
+  describe(`authenticate using implicit grant flow`, () => {
+    const browserSpock = browser.select(`browserSpock`);
+    let spock;
+
+    before(`creates spock`, () => testUsers.create({count: 1, config: {displayName: `Spock`}})
+      .then((users) => {
+        [spock] = users;
+      }));
+
+    step(`browses to sample app and verifies the user is not authenticated`, () => {
+      browserSpock.url(`/packages/node_modules/sample-browser-auth-implicit`);
+      assert.equal(browserSpock.getText(`#authentication-status`), `Not Authenticated`);
+    });
+
+    step(`initiates implicit grant authentication flow`, () => {
+      browserSpock.click(`[title="authenticate"]`);
+    });
+
+    step(`enters user login information`, () => {
+      // Enter Email
+      browserSpock.waitForExist(`input[type=email]`);
```

Comment:
I've switched to using `.waitUntil` because it allows you to provide a failure message. Otherwise the logs just say "promise rejected". Not a blocker, but something to keep in mind while working with webdriver.io